iphone/package.json

```diff
@@ -19,8 +19,8 @@
 	"minIosVersion": "9.0",
 	"minWatchosVersion": "2.0",
 	"vendorDependencies": {
-		"xcode": ">=8.0 <=10.x",
-		"ios sdk": ">=9.0 <=12.x"
+		"xcode": ">=9.0 <=11.x",
```

Comment:
Does raising this and the below `ios sdk` values cause breakage? As I understand it this is used to handle our minimum versions right?